tools/ci_connector_ops/ci_connector_ops/utils.py

```diff
@@ -197,9 +199,17 @@ def name(self):
     def connector_type(self) -> str:
         return self._get_type_and_name_from_technical_name()[0]
 
+    @property
+    def documentation_directory(self) -> Path:
+        return Path(f"./docs/integrations/{self.connector_type}s/{self.name}")
+
     @property
     def documentation_file_path(self) -> Path:
-        return Path(f"./docs/integrations/{self.connector_type}s/{self.name}.md")
+        return self.documentation_directory / README_FILE_NAME
```

Comment:
@erohmensing Im just going ahead on this with the assumption that the existing docs will be ported to the README files. 

Let me know if thats no longer true or if you have a branch I can target :)

Reply:
Here ya go! 

https://github.com/airbytehq/airbyte/pull/27308